src/watchful/enrich.py

```diff
@@ -366,9 +364,7 @@ def main(args: List[str] = None, custom_enricher: Enricher = None) -> None:
     # Upload created attributes output file to Watchful application if it is
     # remote.
     else:
-        client.upload_attributes(
-            dataset_id, dest_attr_filename, args.out_file, project_id
-        )
+        client.upload_attributes(dataset_id, args.out_file)
```

Comment:
Suggest `_ = client.upload_attributes(dataset_id, args.out_file)` so we easily know there is information returned there that may be used in future.

Reply:
@petterik LGTM, just one comment above. Have you run the api tests?

Reply:
Yes! I had some failures but those have been resolved now. I'll merge this and create a new release